storage/src/test/java/tech/pegasys/teku/storage/server/kvstore/InMemoryV4KvStoreBlindedBlocksDatabaseTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright ConsenSys Software Inc., 2022
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.storage.server.kvstore;
+
+import java.io.File;
+import tech.pegasys.teku.storage.server.DatabaseVersion;
+import tech.pegasys.teku.storage.server.StateStorageMode;
+import tech.pegasys.teku.storage.storageSystem.InMemoryStorageSystemBuilder;
+import tech.pegasys.teku.storage.storageSystem.StorageSystem;
+import tech.pegasys.teku.storage.store.StoreConfig;
+
+public class InMemoryV4KvStoreBlindedBlocksDatabaseTest
+    extends AbstractKvStoreDatabaseWithHotStatesTest {
```

Comment:
Don't need to solve it in this PR, but we probably need to find a way to rework these db tests so that we can create variants better - we'll need all of the existing tests with and without blinded blocks since the change applies to older formats as well.